src/api/app/views/webui/request/show.html.erb

```diff
@@ -253,6 +253,8 @@
         <% @my_open_reviews.each_with_index do |review, index| %>
             <div class="review_descision_display <%= 'hidden' if index != 0 %>" id="review_descision_display_<%= index %>">
               <%= form_tag(action: 'modify_review') do %>
+                  <p><%= user_with_realname_and_icon(review[:who], short: true) %> 'requested:'</p>
```

Comment:
The quotes around `requested:` should be removed

![old-ui](https://user-images.githubusercontent.com/1102934/53898678-d48f0600-4038-11e9-9869-05e4485e28d1.png)